sdk/core/core-rest-pipeline/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/core-rest-pipeline",
-  "version": "1.3.3",
+  "version": "1.4.0",
```

Comment:
ah since we're doing a minor bump I feel okay about the default retry count change :)